plugins/CoreHome/Columns/IdSite.php

```diff
@@ -51,4 +57,14 @@ public function onAnyGoalConversion(Request $request, Visitor $visitor, $action)
     {
         return $request->getIdSite();
     }
-}
\ No newline at end of file
+
+    public function formatValue($value, $idSite, Formatter $formatter)
+    {
+        try {
+            $siteName = Site::getNameFor($value);
+            return $siteName . ' (ID ' . $value . ')';
```

Comment:
Do we really need the ID in there? I think it would be much better without.

Reply:
Actually would it be possible to have it as a metadata? If not, we can remove it as it will look better/simpler without.

Reply:
Only way I can think of to add it as metadata would be to:

1. create a new method in base `Dimension` class like `decorateRow($row)` that accepts the row object. (doesn't have to be @api)
2. in CustomReports, call this method before calling `formatValue()`.
3. then in IdSite, add a `decorateRow()` method to set the idsite as row metadata.

If you (@mattab / @tsteur) approve of this approach, then would take me ~30 mins.

Reply:
@diosmosis lets not do this for now. Dimensions should not have any information about rows etc and I would prefer we don't hack anything in there now just to have something there we don't even really need right now etc.

Reply:
Ok, no worries.